### PR TITLE
First pass at Hive data types

### PR DESCRIPTION
**Pull request by edgararuiz**
Based on experience, mapped datetime to "TIMESTAMP" and character  to "STRING"

### CODE REVIEW

File: R/DataTypes.R [hunk 1]
```diff
@@ -77,6 +78,22 @@ odbcDataType.default <- function(info, obj, ...) {
 }
 
 #' @export
+`odbcDataType.Hive` <- function(info, obj, ...) {
+  switch_type(obj,
+    factor = "VARCHAR(255)",
```
Comment (jimhester) [inline R/DataTypes.R]: If character is "STRING" I think factor and list should be as well, as those types are converted to character vectors on the R side before being passed to the ODBC code.

**Comment by jimhester** [comment]
Thanks Edgar, looks good, one minor point above and could you add a note to NEWS.md with the change and mentioning this PR # and yourself `(#38, @edgararuiz)`.

File: R/DataTypes.R (hunk 1 above)
Comment (edgararuiz) [inline R/DataTypes.R]: I thought about that, but wasn't sure how homogenous you'd like to have the type conversion for this function.  I'll make the change

**Comment by jimhester** [comment]
Thanks!

**Pull request merged**